packages/v4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/lists-and-tables.md

```diff
@@ -8,20 +8,33 @@ showTitle: true
 PatternFly offers two components for displaying large data sets: [data list](/documentation/react/components/datalist/) and [table](/documentation/react/components/table/). While they satisfy similar use cases, choosing the correct component to use in your design will be dependent on the type of data you need to display.
 
 Use [data lists](#data-lists) when:
-* A flexible layout is more important than arranging information in a grid
-* You want to include active content like a chart
-* Content displayed may vary between rows
+* A flexible layout is more important than arranging information in a grid.
+* You want to include active content like a chart.
+* Content displayed may vary between rows.
 
 Use [tables](#tables) when:
-* Users will want to consume data as a grid (i.e. structured rows and columns)
-* You want column headers
+* Users will want to consume data as a grid (i.e. structured rows and columns).
+* You want column headers.
 
 ## Data lists
 PatternFly supports several variations of the data list component.
 
-### Standard data list
+### How to use
+Think of each row in a data list as a container for some formatted content. In PatternFly 4, data list rows can accept any [layout](/documentation/react/layouts/bullseye) supported by the design system as long as all rows apply the same layout. Here are some common layouts that may be useful in a data list:
+
+* **[Grid](/documentation/react/layouts/grid):** When you want to display content in a responsive grid.
+* **[Level](/documentation/react/layouts/level):** When you want to justify content evenly over the width of the row.
+* **[Split](/documentation/react/layouts/split):** When you want to distribute content evenly with a main content area in the center.
```

Comment:
Is there any value in also including the flex layout? That wouldn't have a specific layout, it's more of a general layout builder tool, allowing you to create a custom layout.

Reply:
Sure. Any idea about how to describe this?

Reply:
Maybe "When you want a custom layout and more control over the alignment and spacing provided in the other layouts."?